standards/mobile_app_standards.md

```diff
@@ -0,0 +1,135 @@
+
+There are other recommendations to also bear in mind, especially around the security of data on the device and authentication strategies, so you are strongly advised to also read the accompanying **\[guidance\](/guides/mobile\_app\_guidance.md)** on mobile app development alongside these standards.
+
+Which Type of Mobile Technology to Adopt
+----------------------------------------
+
+Mobile apps will usually be required to work on multiple devices and operating systems. For example, for apps intended for general public use, we have already stated that mobile apps created by the DEFRA family must work on at least the Apple and Android platforms. That alone creates a considerable development challenge, and can mean continuing high costs of ownership, especially as those platforms are rapidly changing, and some suppliers, such as Apple, impose additional constraints and demands on users of their technology stack. Add in the need for the mobile app to work readily even with no internet access, and the additional concerns mobile poses to security, and the approach adopted to deliver a mobile app needs to be chosen extremely carefully, following the standard hierarchy explained below, **starting from the top and only working your way further down the options where it is clearly necessary to do so.**
+
+### 
+
+### Option 0: Pick Something "Off-The-Shelf"
+
+This means that, **before developing an in-house app, it is even more important than for other bespoke developments to consider if a Commercial Off-The-Shelf (COTS) approach is preferred**. For example, is a suitable app already available in the app stores? Also, if the app is intended for use with an already adopted COTS suite, for example a time recording or asset management system, then is it possible that the supplier of that suite already has or will soon have a suitable app available, removing the need for us to develop our own app and then integrate that app with third party software? Then not only the development costs of the app, but also the ongoing costs of ownership, will be shared across the supplier's many customers rather than being having to be paid by us alone.
+
+If using a third party app of any kind carefully assess whether use of that app will be appropriate given the sensitivity of the data it will be handling. In particular, many apps, especially free apps available in app stores, harvest user data, for example via Google Analytics. If those data harvesting features cannot be reliably disabled, and the data being processed by the app is in any way sensitive, then the use of such an app would be entirely inappropriate.
+
+### Option 1: Develop a Progressive Web App (PWA)
+
+**Where an in-house app is needed the first option to consider is a Progressive Web App (PWA), as recommended by the Government Digital Service here: <https://www.gov.uk/service-manual/technology/working-with-mobile-technology>**
```

Comment:
PWA is definitely preferred when developing for external users, but for internal there is a good chance that we need to work with internal systems/services so potentially creating a dedicated Power App would be a preferable option compared to building something completely bespoke.

Reply:
Agree power apps would have a place., especially for internal systems using the MS/Azure tech stack. I'll add power apps to the list of options that should be worked through when deciding what to procure, adopt or build.